### ISSUE TITLE
Distance between expanded tool icons in the toolbar

### ISSUE THREAD

**Issue by asemoller** [origo-map/origo#1409]
**Is your feature request related to a problem? Please describe.**
There are starting to be several tools that contain many functions/features as they expand, resulting in many icons on the map. It can be difficult for the user to see which icon belongs to the expanded tool when all icons are at the same distance. It would be nice if it becomes clearer which icons belong together when you expand a tool.

**Describe the solution you'd like**
A more user-friendly interface, for example by reducing the distance between the expanded tool icons or having a frame around them or something else that clarifies which icons belong to the tool.

**Describe alternatives you've considered**
Do nothing and get the user to accept this. But since we have received comments on this, it may be good to review whether we should do something about it or just leave it as it is.

**Additional context**
I really like the round icons on the map so I would at least like to keep that look.

Has anyone else had these thoughts or received any comments on this?

![image](https://user-images.githubusercontent.com/17123002/148770573-9ae818b8-cfca-423e-b07d-3be090a31b1e.png)

**Comment by johnnyblasta** [comment]
The distances between expanded icons are already reduced as far as I can see! Do you want it more reduced? @asemoller

**Comment by steff-o** [comment]
Personally I prefer the edit toolbar style where the currently active control's buttons are in the bottom row. I think that is much clearer.

**Comment by tonnyandersson** [comment]
Given the ever increasing number of tools, perhaps @steff-o is on to something here. Maybe all tools should open in the bottom toolbox.

**Comment by johnnyblasta** [comment]
> Given the ever increasing number of tools, perhaps @steff-o is on to something here. Maybe all tools should open in the bottom toolbox.

Yes, and not just more tools, but also more buttons within the tools.

**Comment by asemoller** [comment]
Yes maybe that's a good idea, but just one thing though, what if you want to have more than one tool active at the same time?

**Comment by tonnyandersson** [comment]
Yep, I guess that's the main drawback.

**Comment by steff-o** [comment]
I haven't tried all controls, but in which case is it possible to have several tools expanded at the same time? At least editor, multiselect and measure closes when another tool is selected.

And when is it necessary to have several tools expanded at the same time? It's only one click away.

**Comment by jokd** [comment]
The developer groups suggestion is to move multiselect and measure tools to the bottom container like draw and editor and that way free up space on the left. These tools are not active at the same time and won't conflict with each other. They will still be activated with buttons on the left side.
